CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+> When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+So you want to be a part of "The Compose Revolution" and that's great. Here are some contribution opportunities, in order of difficulty.
+
+## Documentation & Tutorials
+
+**Difficulty** : Beginner
+
+Go through the docs and tutorials, following the steps in each one. Identify mistakes where things are out of date or don't align with the code or are confusing and submit fixes.
+
+
+## Community Contributions
+
+**Difficulty** : Beginner
+
+Building cool stuff and posting it on twitter and reddit can help inspire other people to join the community and build cool stuff. You use this as an opportunity to learn Compose, find bugs, and occasionally fix bugs too, all at the same time.  And if you build cool things and share them in open source such that people can share/reuse your code, that can help them get started. Getting people excited and building a great community is important for the success of Compose for Desktop.
+
+## Samples
+
+**Difficulty** : Intermediate
+
+ Often the samples will be slightly out of date (using an older release of Compose for Desktop) or will have subtle bugs. Cleaning up such bugs helps make the samples look good.
```

Comment:
The samples themselves may be slightly out of date (using an older release of Compose for Desktop) or could have subtle bugs. Cleaning up these bugs helps keep the samples looking good.

Reply:
Cool. Will update

Reply:
@B1ggDave  Updated via https://github.com/JetBrains/compose-jb/pull/511/commits/59f6636402682a1f8825ccfc0a74c2bc0b8c3813